projects/igniteui-angular/src/lib/banner/banner.component.ts

```diff
@@ -163,6 +163,9 @@ export class IgxBannerComponent implements IToggleView {
         return this.elementRef.nativeElement;
     }
 
+    @HostBinding('class')
+    public cssClass = 'igx-banner-host';
+
```

Comment:
Why do we need this? Can't we directly target the `igx-expansion-panel-body` in the `%igx-banner` placeholder selector?

Reply:
No, because the `.igx-banner` is not on the host element, it is inside the `igx-expansion-panel-body`.

<img width="539" alt="Screenshot 2022-06-30 at 14 56 09" src="https://user-images.githubusercontent.com/21260963/176671612-d9e77839-f3df-4a81-9d60-a2b99d2234e9.png">